docs/context/ref/Action.md

```diff
@@ -0,0 +1,63 @@
+---
+id: Action
+sidebar_label: Action
+title: Action
+hide_title: true
+---
+# `Action`
+
+A representation of an FDC Action (Context or Context+Intent) that can be inserted inside another object, 
+for example a chat message.
+
+## Type
+
+`fdc3.entity.action`
```

Comment:
I think this should be `fdc3.action` to fit in with established naming conventions, particularly as it can be reused outside the context of chat interactions.

```suggestion
`fdc3.action`
```